vars/withGCPEnv.groovy

```diff
@@ -90,33 +90,13 @@ def downloadInstaller(where) {
   def tarball = "gsutil.${isUnix() ? 'tar.gz' : 'zip'}"
 
   dir(where) {
-    if (!downloadWithWget(tarball, url)) {
-      downloadWithCurl(tarball, url)
+    if (!downloadWithWget(url: url, output: tarball)) {
+      downloadWithCurl(url: url, output: tarball)
     }
```

Comment:
this is the implementation of the `download` step 😄

Reply:
:P